packages/engine/Source/DataSources/ModelVisualizer.js

```diff
@@ -445,106 +442,36 @@ ModelVisualizer.prototype.getBoundingSphere = function (entity, result) {
 
   const scene = this._scene;
   const globe = scene.globe;
+  const ellipsoid = defaultValue(globe?.ellipsoid, Ellipsoid.WGS84);
```

Comment:
Did we make a decision on https://github.com/CesiumGS/cesium/issues/11674?

Reply:
Officially no.

If we do go with #11674, I think it should be replaced all at once rather than piecemeal.